doc/guide/cockpit-channel.xml

```diff
@@ -168,7 +168,7 @@ channel.addEventListener("control", function(event, options) { ... })
   <refsection id="cockpit-channels-onready">
     <title>channel.onready</title>
 <programlisting>
-$(channel).on("ready", function(event, options) { ... })
+channel.addEventListener("ready", function(event, options) { ... })
 channel.addEventListener("ready", function(event, options) { ... })
```

Comment:
As lars noted there's now lots of these duplicate lines. While I support removing the jQuery stuff from the docs ... lets not have duplicate lines like this.